misc/config_tools/schema/types.xsd

```diff
@@ -113,13 +113,30 @@ higher value (lower severity) are discarded.</xs:documentation>
   earliest effective virtual time. *TODO: BVT scheduler will be built on
   top of a prioritized scheduling mechanism, i.e. higher priority threads
   get scheduled first, and same priority tasks are scheduled per BVT.*
+- ``SCHED_PRIO``: The priority based scheduler. vCPU scheduling will be based on
+  their pre-configured priorities.
 
 Read more about the available scheduling options in :ref:`cpu_sharing`.</xs:documentation>
   </xs:annotation>
   <xs:restriction base="xs:string">
     <xs:enumeration value="SCHED_NOOP" />
     <xs:enumeration value="SCHED_IORR" />
     <xs:enumeration value="SCHED_BVT" />
+    <xs:enumeration value="SCHED_PRIO" />
+  </xs:restriction>
+</xs:simpleType>
+
+<xs:simpleType name="PriorityType">
+  <xs:annotation>
+    <xs:documentation>Two priorities are supported for priority based scheduler:
+
+- ``PRIO_LOW``: low priority for vCPU scheduling.
+- ``PRIO_HIGH``: high priority for vCPU scheduling.
```

Comment:
My question ties back to what @dbkinder asked earlier but what about `PRIO_IDLE` and `PRIO_MAX`? Aren't these valid options?

Reply:
This section will only appear when the `SCHED_PRIO` scheduler is selected then, is that correct?

Reply:
The PRIO_IDLE and PRIO_MAX are not options for users. They are only for ACRN coding. The values between PRIO_IDLE and PRIO_MAX will expose to users for configuration. 

Yes. A VM need to configure to use either PRIO_LOW or PRIO_HIGH only when it chooses SCHED_PRIO.